app/models/shelter.rb

```diff
@@ -3,7 +3,7 @@ class Shelter < ApplicationRecord
   has_many :pets
   has_many :reviews
 
-  def total
+  def total_pets
```

Comment:
I like this name much better. It more accurately reflects what we want the total of.